helper-cli/src/main/kotlin/commands/TransformResultCommand.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
+
+import com.schibsted.spt.data.jslt.Parser
+
+import org.ossreviewtoolkit.model.yamlMapper
+import org.ossreviewtoolkit.utils.expandTilde
+
+/**
+ * A command to transform an ORT result using a JSLT expression.
+ *
+ * See https://github.com/schibsted/jslt.
+ */
+class TransformResultCommand : CliktCommand(
+    name = "transform", help = "Implements a JSLT transformation on the given ORT result file."
+) {
+    private val ortResultFile by option(
+        "--ort-result-file",
```

Comment:
I think we usually also allow "-i" for the input file.

Reply:
I was thinking the same, but it seems like @fviernau does not use `-i` in the helper-cli commands.

Reply:
Yes, I aligned the arguments with other helper-cli commands, but I agree that a short "-i" alias would be useful. Should I add it?

Reply:
If so, we should do so in a new commit for *all* subcommands. I'd vote for it, but I'd let @fviernau decide.

Reply:
The reason why I decided against short options was because I believe helper cli is mainly used from scripts, not directly by humans. I consider usage of short options from scripts less clean compared to the verbose variant. In short, safe a little maintainance costs is the underlying reason because IMO the added value is very low. 
My preference would be to not add short options to helper cli at all but I wouldn't block it either.

Reply:
> My preference would be to not add short options to helper cli at all

Please note that there already are short options for *some* commands in `master`:

```
$ git grep -- '"-i"' helper-cli/
helper-cli/src/main/kotlin/commands/ImportCopyrightGarbageCommand.kt:        "--input-copyright-garbage-file", "-i",
helper-cli/src/main/kotlin/commands/MapCopyrightsCommand.kt:        "--input-copyrights-file", "-i",
helper-cli/src/main/kotlin/commands/MergeRepositoryConfigurationsCommand.kt:        "--input-repository-configuration-files", "-i",
```

Reply:
So ditching the options would be my preference, but I'll leave it up to you.